Open Judge System/Workers/OJS.Workers.ExecutionStrategies/CSharpProjectTestsExecutionStrategy.cs

```diff
@@ -139,30 +126,59 @@ public override ExecutionResult Execute(ExecutionContext executionContext)
             }
 
             // Delete tests before execution so the user can't access them
-            FileHelpers.DeleteFiles(testPaths.ToArray());
+            FileHelpers.DeleteFiles(this.TestPaths.ToArray());
 
             var executor = new RestrictedProcessExecutor();
             var checker = Checker.CreateChecker(
                 executionContext.CheckerAssemblyName,
                 executionContext.CheckerTypeName,
                 executionContext.CheckerParameter);
 
-            result = this.RunUnitTests(executionContext, executor, checker, result, compilerResult.OutputFile);
+            result = this.RunUnitTests(
+                this.NUnitConsoleRunnerPath,
+                executionContext,
+                executor,
+                checker,
+                result,
+                compilerResult.OutputFile,
+                AdditionalExecutionArguments);
+
             return result;
         }
 
+        protected void WriteSetupFixture(string directory)
```

Comment:
Not sure if `Write` is the most descriptive verb in this situation.  Perhaps use something similar to "Save" or maybe "WriteTo<somewhere>"